istio.io/configmap-nginx.yaml

```diff
@@ -84,6 +84,14 @@ data:
           return 200 "ok\n";
         }
 
+        # Proxy to past versioned instances of the site.
+        location ~ ^/v-[0-9]+\.[0-9]+.*$ {
```

Comment:
Missing:
- `proxy_set_header Host istiodocs-v01.firebaseapp.com`,
- `proxy_set_header Connection ""`,
- `proxy_ssl_name istiodocs-v01.firebaseapp.com`.

Ideally, we should also add whatever SSL CA Firebase is using and verify the SSL certificate.